backend/mora/common.py

```diff
@@ -359,6 +354,13 @@ def create_organisationsfunktion_payload(
         }
     }
 
+    if tilknyttedebrugere:
+        brugere = []
```

Comment:
Minor style nit: I'd use a list comprehension here, e.g.:

```python
org_funk['relationer']['tilknyttedebrugere'] = [
  {
    'uuid': brugerid,
  }
  for brugerid in tilknyttedebrugere
  if brugerid
]
```

This is more readable and more concise.